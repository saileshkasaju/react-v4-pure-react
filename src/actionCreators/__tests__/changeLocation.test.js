import changeLocation from '../changeLocation';

test('changeLocation action creator', () => {
  expect(changeLocation('Seattle, WA')).toMatchSnapshot();
  expect(changeLocation('Seattle, WA')).toEqual({ type: 'SET_LOCATION', payload: 'Seattle, WA' });
});
