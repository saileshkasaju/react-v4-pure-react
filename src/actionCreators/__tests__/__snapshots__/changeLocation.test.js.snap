// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changeLocation action creator 1`] = `
Object {
  "payload": "Seattle, WA",
  "type": "SET_LOCATION",
}
`;
